compiler_opt/rl/corpus.py

```diff
@@ -32,6 +32,8 @@
 # command line, where all the flags reference existing, local files.
 FullyQualifiedCmdLine = Tuple[str, ...]
 
+DEFAULT_CORPUS_DESCRIPTION_FILENAME = 'corpus_description.json'
```

Comment:
_DEFAULT_CORPUS_...

Reply:
_DEFAULT_...

Reply:
I mean adding underscore('_') before DEFAULT_...

github believes that '_'DEFAULT_... means italic the word following the '_'